pkgs/development/tools/sumneko-lua-language-server/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, ninja, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "sumneko-lua-language-server";
+  version = "1.9.0";
+
+  src = fetchFromGitHub {
+    owner = "sumneko";
+    repo = "lua-language-server";
+    rev = "${version}";
+    sha256 = "0d93ikxs0c5fm9k57xbzgphm90dnfqq3nz2bc3bqxi7gzyldb9z6";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    ninja
+    makeWrapper
+  ];
+
+  patches = [
+    ./sumneko_home_expansion.patch
```

Comment:
Did you fetch the from a PR or commit? Would you mind upstreaming it if not?

Reply:
No, I don't know that this can be upstreamed. I'm in correspondence with the author about it